modules/ossm-rn-known-issues.adoc

```diff
@@ -26,14 +26,33 @@ These limitations exist in {ProductName}:
 
 These are the known issues in {ProductName}:
 
-* link:https://issues.redhat.com/browse/MAISTRA-1502[Maistra-1502] As a result of CVEs fixes in version 1.0.10, the Istio dashboards are not available from the *Home Dashboard* menu in Grafana. The Istio dashboards still exist. To access them, click the *Dashboard* menu in the navigation panel and select the *Manage* tab.
+* link:https://issues.redhat.com/browse/OSSM-296[OSSM-296] When adding health configuration to the Kiali custom resource (CR) is it not being replicated to the Kiali configmap.
 
-* link:https://bugzilla.redhat.com/show_bug.cgi?id=1821432[Bug 1821432] Toggle controls in {product-title} Control Resource details page do not update the CR correctly. UI Toggle controls in the Service Mesh Control Plane (SMCP) Overview page in the {product-title} web console sometimes update the wrong field in the resource. To update a SMCP, edit the YAML content directly or update the resource from the command line instead of clicking the toggle controls.
+* link:https://issues.redhat.com/browse/OSSM-289[OSSM-289] In the Kiali console, on the Service Details pages for the 'istio-ingressgateway' and 'jaeger-query' services there are no Traces being displayed. The traces exist in Jaeger.
+
```

Comment:
This should probably (not sure, as it's Tech Preview) also mention [MAISTRA-1947](https://issues.redhat.com/browse/MAISTRA-1947): currently, updates to ServiceMeshExtensions aren't picked up. They have to deleted and re-created.

Reply:
Added to the known issues.